gapis/api/vulkan/extensions/khr_swapchain.api

```diff
@@ -193,7 +205,7 @@ sub bool imageUsedByAnotherSwapchain(
     for _, _, s in Swapchains {
       if (s != null) && (s != swapchainObject) {
         for _, _, i in s.SwapchainImages {
-          if i == imageObject {
+          if i.VulkanHandle == imageObject.VulkanHandle {
```

Comment:
What is the reason for this change?

Reply:
The code used to do a "deep equality" test, whereas we only want to check whether the handles are the same: this is particularly important as if an image handle from a retired swapchain image has been "recycled" for another swapchain image, then the image objects may have the same Vulkan handle, but be different otherwise.

Reply:
I see. Makes sense.